incubator/hnc/internal/validators/object_test.go

```diff
@@ -308,6 +308,20 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Deny creation of object with invalid HNC annotation",
+		fail: true,
+		inst: &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "v1",
+				"kind":       "Pod",
+				"metadata": map[string]interface{}{
+					"annotations": map[string]interface{}{
+						"invalid.hnc.x-k8s.io/select": "foo",
```

Comment:
I'd try a few other example, e.g.:

```
propagate.hnc.x-k8s.io/slct // misspelled part after the slash
hnc.x-k8s.cio/foo // unknown annotation
```

Reply:
+1 and one more plain word annotation key? e.g. "plainword".

Reply:
@yiqigao217 do you mean `plainword.hnc.x-k8s.io/select`?

Reply:
Agreed that the second example (`hnc.x-k8s.io/foo`) is not a part of this change. My apologies for misspelling it in the original exampl!

Reply:
done the testcase with `plainword`